package.json

```diff
@@ -32,15 +32,15 @@
     "clean:build": "gulp cleanBuildDir",
     "clean:release": "gulp cleanReleaseDir",
     "checkin": "gulp checkin",
-    "checkin:built": "gulp checkinBuilt",
     "deployDemos": "gulp deployDemos",
     "deployDemos:beta": "gulp deployDemosBeta",
-    "format": "gulp clangFormat",
+    "format": "gulp format",
     "format:sortrequires": "gulp sortRequires",
     "generate:langfiles": "gulp generateLangfiles",
     "license": "gulp checkLicenses",
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
+    "only:compile:advanced": "gulp onlyBuildAdvancedCompilationTest --debug",
```

Comment:
Not sure I like `only` as a prefix. Would `test:compile:advanced` work?

Reply:
The point is to have `test:compile:advanced` (which is intended to be invoked manually) do all required build steps (in this case: run `tsc` and `closure-make-deps`), but have `only:compile:advanced` (which is intended to be invoked only from `run_all_tests.js`) skip those steps because they'll have just been done in a previous step of the tests.

Luckily, with PR #6431 I think this can be eliminated, since the `test` task should be able to directly invoke the  `buildAdvancedCompilationTest` task.